src/Geometry/Primitives/NonCSG/ConvexPolygon.jl

```diff
@@ -21,7 +21,7 @@ The local frame defines the plane (spans by the right and up vectors) with the p
 the local_polygon_points are given with respect to the local frame and are 2D points.
 NOTE: This class uses static vectors to hold the points which will lead to more efficient performance, but should not be used with polygons with more than 20-30 points.
 """
-struct ConvexPolygon{N,T<:Real}  <: PlanarShape{T} 
+struct ConvexPolygon{T<:Real, N}  <: PlanarShape{T} 
```

Comment:
was there some reason you changed the order of T and N? I did it this way so it would be similar to the StaticArrays type declarations. Does having N first cause problems with the code?

Reply:
reversing this change after our conversation.